cypress-io/cypress
# fix: Display full spec file extension in popovers

- Closes #23064 

### User facing changelog
* Fix issue where popovers would not display full filename of spec files

### Additional details

### Steps to test
1. Open [this example project](https://github.com/cypress-io/cypress/files/9338628/flaky-e2e.zip) e2e project in the app - this project has existing cloud recording data including flaky runs in staging
2. Ensure you are logged in to the dashboard - latest run and flaky data should automatically load
3. Mouse over the Latest Runs row items, validate that the full spec filename is displayed in the popover
4. Find a flaky test, mouse over the Flaky indicator and validate the full spec filename is displayed in the popover

### How has the user experience changed?
**Before** - note the missing ".cy"
![before-latest-runs](https://user-images.githubusercontent.com/11482842/184678019-46a787ac-3c6e-4d44-adcd-bf993962d864.png)

![before-flaky](https://user-images.githubusercontent.com/11482842/184678033-6eb3c6f1-656c-478b-950e-f7e48854304b.png)

**After**
![after-latest-runs](https://user-images.githubusercontent.com/11482842/184678047-6e938bd3-b6a2-4353-b5de-4d23ab4b7a06.png)

![after-flaky](https://user-images.githubusercontent.com/11482842/184678069-d4ee387e-b348-4cb4-ac36-6e22a763dc3e.png)

### PR Tasks
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
